server/plugin/command.go

```diff
@@ -552,6 +553,16 @@ func (p *Plugin) handleWebhookAdd(_ *plugin.Context, parameters []string, args *
 	ctx := context.Background()
 	githubHook, _, err := p.CreateHook(ctx, githubClient, owner, repo, hook)
 	if err != nil {
+		if repo == "" {
```

Comment:
Why is this check needed?

Reply:
@hanzei current scope which we have is already having access to create the repo level so on getting the error while creating hook with org is only scenario where we are out of scope so tried to check on org hook commands.